src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -445,6 +445,22 @@ public bool DisableCustomPackageLocations
         /// Show Run Preview flag.
         /// </summary>
         public bool ShowRunPreview { get; set; }
+
+        /// <summary>
+        /// Limits the size of the tags used by the SearchDictionary
+        /// This static property is not serialized and is assigned NodeSearchTagSizeLimit's value 
+        /// if found at deserialize time.
+        /// </summary>
+        internal static int NodeSearchTagSizeLimitValue = 300;
```

Comment:
this static property is just for easy access from the SearchDictionary.
If I remove it ...I would have to pass along the property `NodeSearchTagSizeLimit` to all instances of SearchDictionary (which means more changes...)